syntax = "proto3";

package mcproto;

message NetworkInfo
{
    uint32 bandwidthUsage = 1;
}